pkg/controllers/plugin/plugin_workload_status_controller.go

```diff
@@ -31,23 +33,49 @@ const (
 	helmReleaseNameAnnotation      = "meta.helm.sh/release-name"
 	helmReleaseNamespaceAnnotation = "meta.helm.sh/release-namespace"
 	jobNameLabel                   = "batch.kubernetes.io/job-name"
-	monitoringAPIGroupName         = "monitoring.coreos.com"
 )
 
-type ReleaseStatus struct {
-	ReleaseName         string `json:"releaseName,omitempty" protobuf:"bytes,1,opt,name=releaseName"`
-	ReleaseNamespace    string `json:"releaseNamespace,omitempty" protobuf:"bytes,3,opt,name=namespace"`
-	HelmStatus          string `json:"helmStatus,omitempty"`
-	ClusterName         string `json:"clusterName,omitempty"`
+var objectFilter = []helm.ManifestObjectFilter{
+	{APIVersion: "v1", Kind: "Pod"},
+	{APIVersion: "v1", Kind: "Deployment"},
+	{APIVersion: "v1", Kind: "StatefulSet"},
+	{APIVersion: "v1", Kind: "DaemonSet"},
+	{APIVersion: "v1", Kind: "ReplicaSet"},
+	{APIVersion: "v1", Kind: "Job"},
+	{APIVersion: "v1", Kind: "CronJob"},
+	{APIVersion: "monitoring.coreos.com/v1", Kind: "Alertmanager"},
+}
+
+var (
+	workloadStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "greenhouse_plugin_workload_status",
```

Comment:
What do you think about adding `up` as a suffix to give the context what 1 and 0 will mean?
```suggestion
			Name: "greenhouse_plugin_workload_status_up",
```